3D/senior-csharp-test/Assets/Cache/FileCacheStorage.cs

```diff
@@ -128,8 +132,7 @@ private List<FileCacheEntry> GetExpiredEntries()
 
 	private void UpdateUseDate( string id )
 	{
-		if( _cacheEntries.ContainsKey( id ) )
```

Comment:
why removing this condition?

Reply:
ah ok when using this function, we know that the id exists

Reply:
Because the method says it will update the use date and its private so we should make sure the entry exist before calling this method (it's actually done). I already removed the check from the Has method last time but got no comment on it so I did the same here.

Reply:
ok for me, I wanted to close this thread but I don't know how to do it

Reply:
Thx , I'll close it :)